frontend/components/organiser/recurring-events/RecurringTemplateDrilldownSidePanel.tsx

```
@@ -42,6 +42,14 @@ const RecurringTemplateDrilldownSidePanel = ({
         >
           Past Events
         </div>
+        <div
+          className={`text-organiser-dark-gray-text font-bold text-md ${
+            currSidebarPage === "Recurrence" ? "bg-white" : ""
+          } hover:bg-white px-8 py-3 transition ease-in-out`}
+          onClick={() => setCurrSidebarPage("Recurrence")}
+        >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)